INSTALL.md

```diff
@@ -0,0 +1,10 @@
+# Install Zync
+
+Download this repository and run `bundle install`.
+
+Install PostgreSQL with `brew install postgresql`
+Start its service with `brew services start postgresql`
+
+Make sure that it is running. See the services running with `brew services list` and `postgresql` should be in that list with status `started` and looking green. If you see an orange `orange` started, that means that it is trying to start but it is having errors, which you can see with more detail in the logs with `tail -f /usr/local/var/log/postgres.log`.
+
+Create the database with `rails db:create`.
```

Comment:
maybe at the top of the INSTALL.md we should have a section with pre-requisites: 
what i see so far: 

* git
* bundler
* brew
* rails

(potentially with links to installation instructions for each?)

Reply:
I don't think this is necessary. `bin/setup` should do everything.

Reply:
@gsaslis `brew` is not a prerequisite. I know this guide is written like it is, but it certainly isn't. Neither is git. You can just get a tar.gz.

Reply:
Yes, good point. We should clarify that the requirements are for *the installation instructions*, not the application itself.

If `bin/setup` replaces all these steps (and doesn’t use them) then no need to mention them ofc.